### DISCUSSION TITLE
2021-11-13 周例会待讨论内容

### DISCUSSION THREAD

**Discussion by liberhe** [Dapp-Learning-DAO/Dapp-Learning#441] [General]
1 国际化（notion, discord活跃， YouTube,  twitter）
  
2 小组学习总结
  comp aave curve总结
  小组学习进度问题：   1 白皮书分享找有一定经验； 2 并行   3 grant，多奖励   

3 职责分工

**Comment by 0x-stan** [comment]
4. 转移贡献不活跃的成员
 跟社区联动

**Comment by liberhe** [comment]
5 邀请国外社区来做技术分享，以及youtube字幕英化
    Bob跟进。graph

**Comment by luckyyang** [comment]
6. 产品讨论
  学习教程

**Reply by luckyyang** [reply]
- 技术产品孵化营
- 赏金平台
- learn to earn
- defi playground（比如用 js），让用户可以去随便用，结合学习的资料，去更深入的了解 defi 的机制
